src/julia_to_gap.jl

```diff
@@ -150,3 +149,6 @@ julia_to_gap(func::Function) = NewJuliaFunc(func)
 
 
 ## TODO: BitArray <-> blist; ranges; ...
+
+# Overload Base.convert
+Base.convert(::Type{T}, x) where T <: GapObj = julia_to_gap(x)
```

Comment:
This seems to go beyond what this PR claims to do, no? There are also no tests at all for this, are there?

Reply:
No tests atm, no. I was to lazy to put the second commit in a single PR. Will do this now.

Reply:
That separate PR with `convert` in it has not yet happened, has it? (Just wondering as I don't see it in the list of open PRs)